### ISSUE TITLE
Update Slack page with rules on spam

### ISSUE THREAD

**Issue by mjang** [writethedocs/www#496]
We've had a couple of spammers in the recent past, and should be specific about our rules for such, in http://www.writethedocs.org/slack/

**Comment by ericholscher** [comment]
Agreed. Spam seems like an obvious thing that isn't allowed, but "commercial posts" in general seem like a better thing to highlight rules around. In general I'd be good with a "no self promotion" rule, except perhaps in a specific channel just for that (eg. #promo or #bragging).

**Comment by plaindocs** [comment]
@Bradamant3 has thoughts on this I think, and maybe even words.

**Comment by Bradamant3** [comment]
My thoughts yesterday were more around other sorts of Slack guidance, but I think today's spam incident highlights a distinction we also need to make, between self promotion (posting a link to something you wrote yourself that might be of general interest), and the Flare consultant posting a link to a blog post he clearly wrote to drive content to the Flare site -- that's a case I'd rather see us keep out altogether. 

TL;DR: promoting your own ideas or thoughts for larger discussion is one thing, promoting your commercial product or service is another (the latter is advertising and if we're going to allow it we should charge for it). 

(There have been some slippery slope instances of the latter that we've let slide bc they haven't involved driving traffic to product sites, but they've already made me and some others a bit queasy. I don't see how we can avoid viral marketing altogether, though.)

**Comment by Bradamant3** [comment]
Whoops, sorry, just saw that this was an issue, not a PR. I'll try to write up something if you like. But Mike seems to have the best radar for this kind of thing :D

**Comment by mjang** [comment]
Maybe I'll make this a writing day task.

**Comment by mxsasha** [comment]
I think this has been sufficiently addressed in #840, and can be closed. Feel free to reopen if I missed something.
